src/main/kotlin/org/rust/ide/annotator/RustItemsAnnotator.kt

```diff
@@ -29,14 +29,16 @@ class RustItemsAnnotator : Annotator {
     }
 
     private fun checkModDecl(holder: AnnotationHolder, modDecl: RustModDeclItemElement) {
-        if (modDecl.isPathAttributeRequired && modDecl.pathAttribute == null) {
+        val pathAttribute = modDecl.pathAttribute
+
+        if (modDecl.isPathAttributeRequired && pathAttribute == null) {
```

Comment:
RustDocAndAttributeOwner.lookupStringValueForKey("path") returns null for this code.
So in this case we still output the old error message.
Maybe it should return "" instead since the attribute is there but empty?
```rust
#[path=""]
mod foo;
```